lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -353,13 +368,13 @@ def refine_lines_centroid(lines, line_width):
 def refine_tracks_centroid(tracks, track_width=None):
     """Refine the tracks based on the brightness-weighted centroid.
 
-    This function interpolates the determined tracks (in time) and then uses the pixels in the vicinity of the
-    tracks to make small adjustments to the estimated location. The refinement correction is
-    computed by considering the brightness weighted centroid.
+    This function interpolates the determined tracks (in time) and then uses the pixels in the
+    vicinity of the tracks to make small adjustments to the estimated location. The refinement
+    correction is computed by considering the brightness weighted centroid.
 
-    *Note: the `track_width` parameter is given in physical units, but the algorithm works with discrete
-    pixels. In order to avoid bias in the result, the number of pixels to use is rounded up to the
-    nearest odd value.*
+    Note: the `track_width` parameter is given in physical units, but the algorithm works with
+    discrete pixels. In order to avoid bias in the result, the number of pixels to use is rounded
+    up to the nearest odd value.
```

Comment:
we could probably turn this into a proper `.. note::`. We've done this in a few other places, would be nice to go through and make things consistent, especially for the rendered public API